src/js/components/help/application-updates/__snapshots__/mender-deb-package.test.js.snap

```diff
@@ -271,16 +140,21 @@ echo "device_type=generic-armv6" | sudo tee /var/lib/mender/device_type
     <span
       style={
         Object {
+          "whiteSpace": "pre-wrap",
           "wordBreak": "break-word",
         }
       }
     >
-      sudo systemctl enable mender && sudo systemctl start mender
+      sudo bash -c 'wget https://d1b0l86ne08fsf.cloudfront.net/master/dist-packages/debian/armhf/mender-client_master-1_armhf.deb && \\
+DEBIAN_FRONTEND=noninteractive dpkg -i mender-client_master-1_armhf.deb && \\
+DEVICE_TYPE="generic-armv6" && \\ 
+mender setup \\
+  --device-type $DEVICE_TYPE \\
+  --demo  && \\
+systemctl restart mender-client'
```

Comment:
Missing `--server-ip` here, or else the command will be interactive. See examples here: https://github.com/lluiscampos/mender-docs/blob/MEN-2427-document-mender-setup-changes/05.Client-configuration/06.Installing/docs.md#unattended-installation

Reply:
This line quoted is from the test snapshot, in the actual js there is a line:
`${ipAddress ? `--server-ip ${ipAddress}` : ''}`
so if a server IP exists, the command **will** have the --server-ip line. But it's not included in that test snapshot, maybe because in the test there won't be an IP?

Reply:
@lluiscampos if that explanation makes sense to you, I'll approve and merge :)

Reply:
It makes sense, thanks.

I also shows why I cannot review GUI code... :sweat_smile: